### DISCUSSION TITLE
Is .Net Framework 4.8 supported by version 3.0.0?

### DISCUSSION THREAD

**Discussion by matteomeucci** [lepoco/wpfui#991] [Q&A] [answered]
When I switch from version 2.1.0 to version 3.0.0 in my .Net Framework 4.8 application nothing works.
UIWinows not exists
UIPgane Not Exists
NavigationItem Not Exists
.....
What's the guidlines to switch from version 2.1.0 to version 3.0.0 using  .Net Framework 4.8?

**Comment by pomianowski** [comment, marked as answer]
Hey @matteomeucci, version 3.0.0 support .NET Framework, but many interfaces have changed.

Newest API can be found here: https://wpfui.lepo.co/api/Wpf.Ui.html
Sample app, here: https://github.com/lepoco/wpfui/blob/main/src/Wpf.Ui.Demo.Mvvm/Wpf.Ui.Demo.Mvvm.csproj
